components/collection/drop/DropConfirmModal.vue

```diff
@@ -0,0 +1,112 @@
+<template>
+  <NeoModal
+    :value="isModalActive"
+    :can-cancel="['outside', 'escape']"
+    scroll="clip"
+    class="top"
+    content-class="modal-width"
+    @close="onClose">
+    <ModalBody :title="$t('drops.finalizeClaimNow')" @close="onClose">
+      <div>
+        <ModalIdentityItem />
+      </div>
+
+      <p class="py-5 is-capitalized">
+        {{ $t('drops.subscribe') }}
+      </p>
+
+      <form @submit.prevent="confirm">
+        <NeoInput
+          ref="emailInput"
+          v-model="email"
+          type="email"
+          required
+          :placeholder="$t('mint.nft.email.placeholder')" />
+
+        <div class="pt-5">
+          <NeoCheckbox v-model="agree" class="is-capitalized">
+            {{ $t('drops.consent') }}
+          </NeoCheckbox>
+        </div>
+
+        <div class="is-flex is-justify-content-space-between pt-4">
+          <NeoButton
+            class="is-flex is-flex-1 btn-height is-capitalized shine"
+            :disabled="disabled"
+            no-shadow
+            variant="k-accent"
+            native-type="submit">
+            {{ submitButtonText }}
+          </NeoButton>
+        </div>
+      </form>
+    </ModalBody>
+  </NeoModal>
+</template>
+
+<script setup lang="ts">
+import { NeoButton, NeoCheckbox, NeoInput, NeoModal } from '@kodadot1/brick'
+import ModalBody from '@/components/shared/modals/ModalBody.vue'
+import ModalIdentityItem from '@/components/shared/ModalIdentityItem.vue'
+
+const emit = defineEmits(['confirm', 'completed', 'close'])
+const { $i18n } = useNuxtApp()
+const props = defineProps<{ modelValue: boolean }>()
+
+const isModalActive = useVModel(props, 'modelValue')
+
+const emailInput = ref()
+const email = ref()
+const agree = ref(false)
+
+const invalidEmail = computed(() => !emailInput.value?.checkHtml5Validity())
+
+const disabled = computed(
+  () => invalidEmail.value || email.value === '' || !agree.value,
+)
+
+const submitButtonText = computed(() => {
+  if (invalidEmail.value) {
+    return $i18n.t('drops.enterValidEmail')
+  }
+
+  if (!agree.value) {
+    return $i18n.t('drops.agreeToProceed')
+  }
+
+  return $i18n.t('drops.subscribeAndClaim')
+})
+
+const onClose = () => {
+  emit('close')
+}
+
+const confirm = () => {
+  emit('confirm', { email: email.value })
+  email.value = ''
+}
+</script>
+
+<style lang="scss" scoped>
+@import '@/assets/styles/abstracts/variables';
+@import '@/assets/styles/abstracts/animations';
+
+.modal-width {
+  width: 25rem;
+}
+
+.btn-height {
+  height: 3.5rem;
+}
+
+.shine {
+  &:not(:disabled) {
+    @include ktheme() {
+      @include shineEffect(theme('k-accentlight3'), lightgrey);
+      &:hover {
+        color: theme('k-accent') !important;
+      }
+    }
+  }
+}
```

Comment:
put the variable in the `tailwind.scss` instead. and `tailwind.config.js`. then:

```suggestion
.shine {
  &:not(:disabled) {
    @include shineEffect(var(--k-accent-light-3), lightgrey);
    &:hover {
      color: var(--k-accent) !important;
    }
  }
}
```

Reply:
fixed